apps/andi/test/unit/andi/input_schemas/dataset_input_test.exs

```diff
@@ -165,6 +169,21 @@ defmodule Andi.InputSchemas.DatasetInputTest do
       assert changeset.errors
              |> Enum.any?(fn {:schema, {error, _}} -> String.match?(error, ~r/selector.+another_field/) end)
     end
+
+    data_test "requires #{field} to have an acceptable value" do
```

Comment:
This test is doing the opposite - it's testing that they can't have an invalid value

Reply:
Okay, we can fix the name of the test.